content/english/facts/soy-is-bad-for-you.md

```diff
@@ -0,0 +1,223 @@
+> However, the NTP accepts the conclusions of the expert panel that the current literature in laboratory animals is limited in its utility for reaching conclusions for infants fed soy infant formula. The NTP agrees with the expert panel that the individual isoflavone studies [...] in laboratory animals would benefit from data on the effects of mixtures of isoflavones and/or other components present in soy infant formula because these mixture studies would better replicate human infant exposures.  
+> - [Soy Infant Formula of Minimal Concern - 2010](https://web.archive.org/web/20191002190225/https://ntp.niehs.nih.gov/ntp/ohat/genistein-soy/soyformula/soymonograph2010_508.pdf)
+
+#### Exposure to Soy-Based Formula in Infancy and Endocrinological and Reproductive Outcomes in Young Adulthood - 2001
+> No statistically significant differences were observed between groups in either women or men for more than 30 outcomes. However, women who had been fed soy formula reported slightly longer duration of menstrual bleeding [...], with no difference in severity of menstrual flow. They also reported greater discomfort with menstruation (unadjusted relative risk for extreme discomfort vs no or mild pain[...]).  
+- [Exposure to Soy-Based Formula in Infancy and Endocrinological and Reproductive Outcomes in Young Adulthood - 2001](https://web.archive.org/web/20191002190445/https://jamanetwork.com/journals/jama/fullarticle/194106)
+
+### Overall
+
+#### Soy Isoflavones: A Safety Review - 2003
+
+>  Whereas results in some studies are limited or conflicting, when viewed in its entirety, the current literature supports the safety of isoflavones as typically consumed in diets based on soy or containing soy products.  
+> - [Soy Isoflavones: A Safety Review - 2003](http://web.archive.org/web/20191002184755/https://www.researchgate.net/profile/Melody_Harwood2/publication/10855804_Soy_Isoflavones_A_Safety_Review/links/5bd721a8a6fdcc3a8dae5fec/Soy-Isoflavones-A-Safety-Review.pdf?origin=publication_detail)
+
+#### Soy, Soy Foods and Their Role in Vegetarian Diets - 2018
+
+This article became one of our primary sources due to its recency and broad coverage of other articles.
```

Comment:
Yes it is an article... though i'm not sure this doe justice to a meta-review of available literature.  Just makes it sound a little too like something from woman weekly.